bin/docker_setup

```diff
@@ -62,5 +62,6 @@ Dir.chdir APP_ROOT do
   run "docker-compose run --rm web rake dev:prime"
   # Create all parallell test databases
   run "docker-compose run --rm web rake parallel:setup"
```

Comment:
Do you think it's worth trying to combine some of these lines into `&&`s like in the `Dockerfile`? Just to minimize docker run overhead, not to like create fewer layers

Reply:
Yeah, I think that's def. an improvement we could make.  I sorta followed the style of the old setup script.  I'll put it on my list to file another PR with these changes

Reply:
Sounds good!